--- a/configure.ac
+++ b/configure.ac
@@ -7129,7 +7129,7 @@
 
 version=$PACKAGE_VERSION
 
-copyright="Copyright (C) 2025 Free Software Foundation, Inc."
+copyright="Copyright (C) 2026 Free Software Foundation, Inc."
 AC_DEFINE_UNQUOTED([COPYRIGHT], ["$copyright"],
   [Short copyright string for this version of Emacs.])
 AC_SUBST([copyright])
@@ -7607,7 +7607,7 @@
 
 AH_TOP([/* GNU Emacs site configuration template file.
 
-Copyright (C) 1988-2025 Free Software Foundation, Inc.
+Copyright (C) 1988-2026 Free Software Foundation, Inc.
 
 This file is part of GNU Emacs.
 
