--- a/configure.ac	2019-08-22 07:25:23.000000000 +0900
+++ b/configure.ac	2020-05-07 13:00:00.000000000 +0900
@@ -23,6 +23,6 @@ dnl  along with GNU Emacs.  If not, see 
 
 AC_PREREQ(2.65)
 dnl Note this is parsed by (at least) make-dist and lisp/cedet/ede/emacs.el.
-AC_INIT(GNU Emacs, 26.3, bug-gnu-emacs@gnu.org)
+AC_INIT(GNU Emacs, 26.3.52, bug-gnu-emacs@gnu.org)
 
 dnl Set emacs_config_options to the options of 'configure', quoted for the shell,
 dnl and then quoted again for a C string.  Separate options with spaces.