--- a/configure.ac	2021-12-02 05:37:08.000000000 +0900
+++ b/configure.ac	2021-12-30 15:56:12.000000000 +0900
@@ -5439,7 +5440,7 @@ fi

 version=$PACKAGE_VERSION

-copyright="Copyright (C) 2021 Free Software Foundation, Inc."
+copyright="Copyright (C) 2022 Free Software Foundation, Inc."
 AC_DEFINE_UNQUOTED(COPYRIGHT, ["$copyright"],
   [Short copyright string for this version of Emacs.])
 AC_SUBST(copyright)
@@ -5830,7 +5831,7 @@ AC_SUBST(WINDOW_SYSTEM_OBJ)

 AH_TOP([/* GNU Emacs site configuration template file.

-Copyright (C) 1988, 1993-1994, 1999-2002, 2004-2021
+Copyright (C) 1988, 1993-1994, 1999-2002, 2004-2022
   Free Software Foundation, Inc.

 This file is part of GNU Emacs.